### DISCUSSION TITLE
Welcome to alfa-leetcode-api Discussions!

### DISCUSSION THREAD

**Discussion by alfaarghya** [alfaarghya/alfa-leetcode-api#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RodEsp** [comment]
👋🏼 Hello!

I run [RodEsp/daily-leetcode-bot](https://github.com/RodEsp/daily-leetcode-bot) which uses the Leetcode API to post the daily problem to a Zulip stream and a Slack channel.

It uses your API as a fallback [here](https://github.com/RodEsp/daily-leetcode-bot/blob/f9687453f8af17f8d9a37da3f1a0d999a78de21a/bot.js#L60-L63). I've found this very useful when Cloudflare's bot detection blocks the bot from accessing Leetcode's API directly.

**Reply by alfaarghya** [reply]
That's Great 😊

**Comment by karthikchunduru** [comment]
Hi Thanks for the API's
I have a small doubt i want to try out this
Contest details --> /contest/:contestSlug

may i know what is contestSlug

i am trying out like this https://alfa-leetcode-api.onrender.com/contest/weekly-contest-399 but its not working please correct me if i am wrong thanks

**Comment by Laurenyoo** [comment]
How often does the data refresh? Its not real time updated right?

**Reply by alfaarghya** [reply]
as soon as the leetcode update their database based on timezone
